### PR TITLE
Typescript

### PR DESCRIPTION
**Pull request by marocchino**
ref riseshia/gannbaruzoi#35

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -40,6 +33,21 @@ new Vue({
                     }
                   }
                 }`
-    }
+} as Query;
```
Comment (riseshia) [inline src/main.ts]: `;`

File: src/main.ts [hunk 2]
```diff
@@ -40,6 +33,21 @@ new Vue({
...
+(<any>window).__APOLLO_CLIENT__ = apolloClient;
```
Comment (riseshia) [inline src/main.ts]: `;`

**Comment by riseshia** [comment]
is there any plan how handle graphql? ;_;

**Comment by marocchino** [comment]
We can use this structure: https://github.com/vuejs/vuex/tree/dev/examples/shopping-cart
- api
- components
- store

**Comment by riseshia** [comment]
meaning to say, not using network interface, but using via store of flux?

**Comment by marocchino** [comment]
yes.

**Review by riseshia — APPROVED** [review]
I got it XD

**Pull request merged**